app/Models/GenereRepository.php

```
@@ -208,12 +234,24 @@ public function cascadeDelete(int $id): bool
+            $this->bindIntParams($stmt, $ids);
+            if (!$stmt->execute()) {
+                throw new \RuntimeException('Errore nella rimozione delle posizioni dei generi');
+            }
+
             $stmt = $this->db->prepare("DELETE FROM generi WHERE id = ?");
             $stmt->bind_param('i', $id);
             $result = $stmt->execute();
 
             $this->db->commit();
-            QueryCache::clearByPrefix('genre_tree_');
+            ContentCache::booksChanged();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\ContentCache' in method 'cascadeDelete'. (undefined)

(StaticAccess)